iron-component-page.html

```diff
@@ -317,7 +317,7 @@
            */
           transitive: {
             type: Boolean,
-            value: false
+            value: true
```

Comment:
who's relying on the default value of false? Will they really "break" if more complete docs are shown in the component page?